solutions/swb-reference/integration-tests/tests/isolated/sshKeys/get.test.ts

```diff
@@ -3,60 +3,43 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import ClientSession from '../../../support/clientSession';
-import Setup from '../../../support/setup';
+import { PaabHelper, PaabResources } from '../../../support/complex/paabHelper';
 import HttpError from '../../../support/utils/HttpError';
-import RandomTextGenerator from '../../../support/utils/randomTextGenerator';
 import { checkHttpError } from '../../../support/utils/utilities';
 
 describe('Get Key Pair negative tests', () => {
```

Comment:
nit (that I should have pointed out earlier): this is no longer a true get key pairs test file since we are hitting the GET list API. Go ahead and rename the file to be `listUserSshKeysForProject.test.ts` and change the describe to reflect the actual method we are hitting. This will just make it clearer to anyone not familiar with this code that this isn't a "get" (singular item) method but a special list method.